iati/core/rulesets.py

```diff
@@ -1,45 +1,841 @@
-"""A module containg a core representation of IATI Rulesets.
+"""A module containing a core representation of IATI Rulesets.
 
 Note:
     Rulesets are under-implemented since it is expected that their implementation will be similar to that of Codelists, which is currently unstable. Once Codelist stability has been increased, Rulesets may be fully-implemented.
 
-Warning:
-    The contents of this module have not been implemented. Their structure may change when they are implemented.
-
 Todo:
-    Implement Rulesets (and Rules). Likely worth completing the Codelist implementation first since the two will be similar.
+    Review for edge cases.
+    Consider how we should handle lxml errors.
+    Remove references to `case`.
+
 """
+from datetime import datetime
```

Comment:
Move to after `sre_constants`

Reply:
See https://github.com/IATI/iati.core/pull/45#discussion_r136947974

Reply:
done